src/charts/bar.js

```diff
@@ -178,7 +184,9 @@ define(function(require) {
                 drawAxis();
 
                 if (enableLabels) {
-                    drawLabels();
+                    localeRequest
+                        .then(() => drawLabels())
```

Comment:
Wouldn't the locale change also the axis tick text?

Reply:
I'm not sure to be honest.

Reply:
No prob, I will test it later.